blackbox.yml

```diff
@@ -60,3 +60,10 @@ modules:
     timeout: 5s
     icmp:
       ttl: 5
+  http_3xx:
+    prober: http
+    http:
+      preferred_ip_protocol: "ip4"
+      enable_http3: true
+      enable_http2: false
```

Comment:
do you always need to disable `enable_http2` when using `enable_http3`? or this is the a example of using http3 only?

thinking about it, I can't think of the case on when you need `enable_http3` and `enable_http2` both? but I might be wrong here.

if it doesn't make sense to have both enabled? let's add a validation so folks are not allowed to enable both?

Reply:
also, let's add the newline back to the file.